_webi/bootstrap.sh

```diff
@@ -182,12 +182,12 @@ function __webi_main () {
         echo ""
     }
 
-    if [[ \$# -eq 0 ]] || [[ "\$1" =~ ^(-V|--version|version)$ ]]; then
+    if [ \$# -eq 0 ] || [ "\$1" =~ ^(-V|--version|version)$ ]; then
```

Comment:
I don't believe that =~ is available in Posix.

Probably something like `echo "\$1" | grep -q '^(-V|--version|version)$'`